modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -23,33 +25,47 @@ spec:
   pipelines:
     - inputRefs: [ myAppLogData ] <3>
       outputRefs: [ default ] <4>
-  inputs:
+  inputs: <5>
     - name: myAppLogData
       application:
         selector:
           matchLabels:
-            environment: production <5>
-            app: nginx <5>
-        namespaces: <6>
+            environment: production <6>
+            app: nginx <6>
```

Comment:
I am not sure <6> adjacent to `environment: production` and `app: nginx` is needed, based on the explanations to the call outs below. We probably need to confirm with the dev team.